app/assets/stylesheets/articles.scss

```diff
@@ -4,297 +4,74 @@
-  margin: auto;
+  min-height: 440px; // TODO: why?
+  margin: 0 auto;
   max-width: 1250px;
   text-align: left;
-  @media screen and (min-width: 950px) {
-    margin-top: 26px;
-  }
-  &.sub-home {
-    margin-top: 10px;
-    .side-bar {
-      @media screen and (min-width: 950px) {
-        margin-top: 12px;
-      }
-    }
-  }
-  .query-results-nothing {
-    padding: 40px 0px;
-    font-size: 1.2em;
-    @include themeable(background, theme-container-background, white);
-    @include themeable(box-shadow, theme-container-box-shadow, $bold-shadow);
-    border-radius: 3px;
-    @include themeable(
-      border,
-      theme-container-border,
-      1px solid $outline-color
-    );
-    width: 94%;
-    margin: auto;
-    margin-top: 12px;
-    @media screen and (min-width: 950px) {
-      width: 100%;
-    }
-  }
 
   .articles-list {
-    text-align: left;
     margin: auto;
-    text-align: center;
-    width: 100%;
+    max-width: 100%;
     position: relative;
     -webkit-tap-highlight-color: rgba(0, 0, 0, 0); //Invisible black
-    max-width: 750px;
-    padding-bottom: 30px;
-    @media screen and (min-width: 750px) {
-      margin-bottom: 30px;
-    }
+    box-sizing: border-box;
+
     @media screen and (min-width: 950px) {
-      width: 100%;
-      margin-left: 2.3%;
-      margin-right: 15px;
-      max-width: calc(100% - 380px);
       float: left;
-    }
-    @media screen and (min-width: 1120px) {
-      max-width: calc(100% - 640px);
-    }
-    @media screen and (min-width: 1240px) {
       width: 100%;
-      margin-left: 2.3%;
-      margin-right: 15px;
-      max-width: 604px;
-      float: left;
-    }
-    .notifications-mobile-filters {
-      padding-bottom: 8px;
-      @media screen and (min-width: 950px) {
-        display: none;
-      }
-      .organization-name {
-        font-size: 15px;
-        font-weight: bold;
-        border: 0px;
-        display: inline-block;
-        width: 27%;
-        padding: 9px 0px;
-        text-align: center;
-        @include themeable(color, theme-secondary-color, $medium-gray);
-        margin: 1%;
-      }
-      .query-filter-button {
-        font-size: 12px;
-        font-weight: bold;
-        background: transparent;
-        border: 0px;
-        display: inline-block;
-        width: 27%;
-        padding: 9px 0px;
-        text-align: left;
-        border-radius: 100px;
-        @include themeable(color, theme-color, $black);
-        text-align: center;
-        margin: 1%;
-        &:hover {
-          @include themeable(
-            background,
-            theme-container-background-hover,
-            lighten($bold-blue, 38%)
-          );
-        }
-        &.selected {
-          @include themeable(
-            background,
-            theme-container-accent-background,
-            lighten($bold-blue, 8%)
-          );
-          @include themeable(color, theme-color, white);
-        }
-      }
+      max-width: calc(100% - 280px);
```

Comment:
Do we know what the `280px` is? Is it potentially just margin `140px` on each side?

Reply:
It's the sidebar width. Previously it was some more/less random. I unified it to be 280px on both sidebars for now. This part of code basically defines the layout of the page and there's lots of things that could be improved (go with flexbox/grid instead of floats, isolate layout-defining properties to be separated from rest of the code, be smarter in terms actually defining all these sizes).